specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/azureFirewall.json

```diff
@@ -537,7 +551,6 @@
           "description": "The firewallPolicy associated with this azure firewall."
         },
         "hubIpAddresses": {
-          "readOnly": true,
```

Comment:
why is 'readonly' removed?

Reply:
I would also like to know.

---
In reply to: [413481713](https://github.com/Azure/azure-rest-api-specs/pull/9151#discussion_r413481713) [](ancestors = 413481713)